src/tests/Test1/test1.verifySorting.spec.js

```diff
@@ -0,0 +1,28 @@
+/* eslint-disable indent */
+import { expect } from '@playwright/test';
+import { test } from '../../fixtures/base';
+
+test.describe('Test1 => Perform and verify sorting on the Inventory page', () => {
+    test('should login successfully, perform and verify sorting', async (
+        /** @type {{ app: import('../pages/Application').Application }} */{ app },
+    ) => {
+        await app.login.navigate();
+        await app.login.performLogin('standard_user', 'secret_sauce');
+
+        await expect(app.inventory.headerTitle).toBeVisible();
+
+        expect(await app.inventory.inventoryItems.count()).toBeGreaterThanOrEqual(1);
+
+        const originalPrices = await app.inventory.itemPrice.evaluateAll(
+            items => items.map(item => parseFloat(item.textContent.replace('$', '')))
+        );
+
+        await app.inventory.selectOption('lohi');
+
+        const sortedPrices = await app.inventory.itemPrice.evaluateAll(
+            items => items.map(item => parseFloat(item.textContent.replace('$', '')))
+        );
+
+          expect(sortedPrices).toEqual([...originalPrices].sort((a, b) => a - b));
```

Comment:
Let`s create a helper function to sort the items, fr example

```
  async expectSortedProducts(sortBy, productNames, productPrices) {
    switch (sortBy) {
      case 'Name (A to Z)':
        const sortedNamesAZ = await this.getProductsNames();
        const expectedNamesAZ = productNames.sort();
        expect(sortedNamesAZ).toEqual(expectedNamesAZ);
        break;
      case 'Name (Z to A)':
        const sortedNamesZA = await this.getProductsNames();
        const expectedNamesZA = productNames.sort().reverse();
        expect(sortedNamesZA).toEqual(expectedNamesZA);
        break;
      case 'Price (low to high)':
        const sortedPricesLowHigh = await this.getProductPrices();
        const expectedPricesLowHigh = productPrices.sort((a, b) => a - b);
        expect(sortedPricesLowHigh).toEqual(expectedPricesLowHigh);
        break;
      case 'Price (high to low)':
        const sortedPricesHighLow = await this.getProductPrices();
        const expectedPricesHighLow = productPrices.sort((a, b) => b - a);
        expect(sortedPricesHighLow).toEqual(expectedPricesHighLow);
        break;
      default:
        throw new Error('Sort option is not correct');
    }
  }
```

Reply:
You should write tests for all sort options

Reply:
done